include/fastdds/dds/xtypes/dynamic_types/DynamicTypeBuilderFactory.hpp

```diff
@@ -148,7 +148,7 @@ class DynamicTypeBuilderFactory : public std::enable_shared_from_this<DynamicTyp
     /*!
      * Creates a new @ref DynamicTypeBuilder reference by parsing the type description at the given URL.
      * @remark Not implemented yet.
-     * @param[in] URL pointing to the XML description.
+     * @param[in] document_url pointing to the XML description.
```

Comment:
```suggestion
     * @param[in] document_url pointing to the url containing the type description.
```

Reply:
Fixed in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)